packages/mantle/src/components/input/input.tsx

```diff
@@ -138,10 +138,10 @@ const InputContainer = ({
 				aria-disabled={disabled ?? _ariaDisabled}
 				data-validation={validation || undefined}
 				className={cx(
-					"h-11 text-base sm:h-9 sm:text-sm",
+					"pointer-coarse:text-base h-11 text-sm",
```

Comment:
```suggestion
					"pointer-coarse:text-base h-9 text-sm",
```
Though the proportions aren't as good on mobile with the larger text, it's more critical that buttons and inputs maintain the same height on desktop layouts (at least). I think it's the right trade-off for now to avoid the iOS zoom.

Reply:
Nice catch, and the whole point of the PR lol. Addressed!